lib/Data/Validate/Sanctions.pm

```diff
@@ -52,7 +52,17 @@ sub update_data {
 sub last_updated {
     my $self = shift;
     my $list = shift;
-    return $list ? $self->{_data}->{$list}->{updated} : $self->{last_time};
+    
+    if ($list) {
+        return $self->{_data}->{$list}->{updated};
+    } else {
+        $self->_load_data();
+        return max (
+        	map ( 
```

Comment:
our usual coding guidelines are to use the `map BLOCK` form, rather than `map EXPR`.

Also, you don't need the keys here - just

```
map { $_->{updated} } values %{$self->{_data}}
```